src/pkg/Microsoft.DotNet.CoreRT.Targets/Microsoft.DotNet.CoreRT.Targets.pkgproj

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This appears to be a package that is no longer produced, even on master.  I believe this was previously runtime.<platform>-<rid>.Microsoft.TargetingPack.Private.CoreRT.  If it sounds correct that this isn't produced, I'll remove this pkgproj.